adapters/tappx/tappx.go

```diff
@@ -126,7 +127,9 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 		}
 	}
 
-	thisURI.Path += params.Endpoint
+	if !(strings.Contains(strings.ToLower(thisURI.Host), strings.ToLower(params.Endpoint))) {
```

Comment:
Could you please add a new json test for this condition?

Reply:
Hi! I have add now the commit of a new json test that is think for this condition.
Also, the idea is when all our partners change to new system, we will update all tests and remove the lines (130-132) that don't be needed more.
Thanks for all!

Reply:
Nice. Thank you.